Some updates to the Windows Insider Program documentation (#247)
MicrosoftDocs/windows-insider: wip/at-home/whats-new-apps.md

Note: 

> * This feature will gradually roll out to Insiders  so it may take a few days to be available inside the Your Phone app.

Reviewer: ```suggestion
* This feature will gradually roll out to Insiders so it may take a few days to be available inside the Your Phone app.
```

> * To enable the Calls feature, users must sign into the  Your Phone app using the Microsoft account that they used to register as a Windows Insider. 

Reviewer: ```suggestion
* To enable the Calls feature, users must sign into the Your Phone app using the Microsoft account that they used to register as a Windows Insider. 
```

> * For set-up instructions and other details see [Calls Feature for the Your phone app](https://answers.microsoft.com/en-us/insider/forum/all/calls-feature-for-the-your-phone-app/8266e234-ba16-42be-8b11-af2e33ba93ed) 

Reviewer: ```suggestion
* For set-up instructions and other details, see [Calls Feature for the Your phone app](https://answers.microsoft.com/en-us/insider/forum/all/calls-feature-for-the-your-phone-app/8266e234-ba16-42be-8b11-af2e33ba93ed).
```

Known issue:

> * In certain instances, Calls feature may require you to pair your mobile phone & PC again. If you have previously paired your devices, unpair your devices and go through the Calls set up flow again.

Reviewer: ```suggestion
* In certain instances, Calls feature may require you to pair your mobile phone & PC again. If you have previously paired your devices, unpair your devices and go through the Calls setup flow again.
```
